src/main/java/frc/robot/RobotContainer.java

```diff
@@ -38,22 +42,18 @@
  * subsystems, commands, and trigger mappings) should be declared here.
  */
 public class RobotContainer {
-  private final DriveSubsystem drive = new DriveSubsystem();
   private final ScoringLocationUtil scoreLoc = new ScoringLocationUtil();
   public final Lift lift = new Lift(scoreLoc);
+  private final DriveSubsystem drive = new DriveSubsystem(lift::throttleForLift);
   public final Intake intake = new Intake(lift::clearOfIntakeZone);
-  private final IntakeLimelight intakeLimelight =
-      new IntakeLimelight(
-          Constants.INTAKE_LIMELIGHT_PITCH_DEGREES,
-          Constants.INTAKE_LIMELIGHT_HEIGHT_METERS,
-          Constants.INTAKE_TARGET_HEIGHT_METERS);
-  private final TagLimelight tagLimelight =
-      new TagLimelight(
-          Constants.TAG_LIMELIGHT_PITCH_DEGREES,
-          Constants.TAG_LIMELIGHT_HEIGHT_METERS,
-          Constants.TAG_TARGET_HEIGHT_METERS);
+  //   private final IntakeLimelight intakeLimelight =
+  //       new IntakeLimelight(
+  //           Constants.INTAKE_LIMELIGHT_PITCH_DEGREES,
+  //           Constants.INTAKE_LIMELIGHT_HEIGHT_METERS,
+  //           Constants.INTAKE_TARGET_HEIGHT_METERS);
```

Comment:
Don't merge commented out code

Reply:
We're gonna want this back so I'm keeping it